x-pack/solutions/observability/plugins/apm/test/scout/ui/parallel_tests/settings/agent_configuration.spec.ts

```diff
@@ -0,0 +1,92 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { expect } from '@kbn/scout-oblt';
+import { test } from '../../fixtures';
+
+test.describe('Agent Configuration - Viewer', { tag: ['@ess'] }, () => {
+  test.beforeEach(async ({ browserAuth }) => {
+    await browserAuth.loginAsViewer();
+  });
+
+  test('shows create button as disabled', async ({ pageObjects: { agentConfigurationsPage } }) => {
+    await agentConfigurationsPage.goto();
+    const createButton = await agentConfigurationsPage.getCreateConfigurationButton();
+    await expect(createButton).toBeDisabled();
+  });
+});
+
+test.describe('Agent Configuration - Privileged User', { tag: ['@ess'] }, () => {
+  test.beforeEach(async ({ browserAuth }) => {
+    await browserAuth.loginAsPrivilegedUser();
+  });
+
+  test('shows create button as enabled', async ({ pageObjects: { agentConfigurationsPage } }) => {
+    await agentConfigurationsPage.goto();
+    const createButton = await agentConfigurationsPage.getCreateConfigurationButton();
+    await expect(createButton).toBeEnabled();
+  });
```

Comment:
I think we should avoid this kind of test, this is more like a unit test in an e2e mode. If the button is not enabled the next test will fail, so I'd remove this one and only keep the next one.